### PR TITLE
Some updates

### PR DESCRIPTION
**Pull request by ligangty**
* Update indy-model to 1.4-SNAPSHOT for new model usage
  * Replace content-browse model class usage with indy-model deps
  * Replace koji model class usage with indy-model deps

### CODE REVIEW

**Comment by ligangty** [comment]
This PR depends on https://github.com/Commonjava/indy-model/pull/31, so will re-check this after that one merging.

**Comment by ligangty** [comment]
This is just some deps modification, so merged directly.

File: pom.xml [hunk 1]
```diff
@@ -87,11 +98,6 @@
         <version>3.3.1-SNAPSHOT</version>
       </dependency>
```
Comment (yma88) [inline pom.xml:99]: This needs to be removed?

Comment (ligangty) [reply pom.xml:99]: Yes. Will do new PR to remove

**Pull request merged**